src/App.tsx

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
-import './App.css';
-import ModalComp from './Components/ModalComp';
+//import './App.css';
+//import ModalComp from './Components/ModalComp';
 import 'antd/dist/antd.css';
+import Filter from './Filter/Filter'
+import './Filter/Filter.css'
 
 
 function App() {
   return (
-      <ModalComp />
+      // <ModalComp />
```

Comment:
Remove comments not needed

Reply:
But this is first task component. So now this is not needed??

Reply:
Removed unwanted lines.